config/routes.rb

```diff
@@ -13,6 +13,8 @@ def draw(routes_name)
     collection do
       get :dashboard
       get :problematic_projects
+      get :award_badges
```

Comment:
why did you move it to organizers controller? can we move it back to separate controller?

Reply:
Well I moved it to organiser controller because the , Controller was just doing one functionality. Also it was organiser task to award badge, to so I moved the award_function here. 
Also, only organisers should be able to see this page, so I thought of moving everything.

Reply:
@pg20 I think it's too much logic in one controller now - awarding badges is separate thing and can be easily separated (just like you did before) so I'd be for moving it back to separate controller. Also I sent you line of code few days back (I think you asked for it on slack) which checks whether logged user is organizer to allow only organizer to access page

Reply:
Yes you gave the link of organiser controller, but i misinterpreted. So I added here :(. Ill move everything back by today.